docs/usage/utils/subscribeToTransactionsWithProofs.md

```diff
@@ -10,8 +11,35 @@ Parameters:
 | **bloomFilter.nTweak**     | Number           | yes            | A random value to add to the seed value in the hash function used by the bloom filter. |
 | **bloomFilter.nFlags**     | Number           | yes            | A set of flags that control how matched items are added to the filter. |
 | **options.fromBlockHash**  | String           | yes            | Specifies block hash to start syncing from |
-| **options.fromBlockHeight**| String           | yes            | Specifies block height to start syncing from |
+| **options.fromBlockHeight**| Number           | yes            | Specifies block height to start syncing from |
 | **options.count**          | Number           | no (default: 0)| Number of blocks to sync, if set to 0 syncing is continuously sends new data as well |
 
 Returns : Promise<EventEmitter>|!grpc.web.ClientReadableStream<!TransactionsWithProofsResponse>
 
+Example :
+
+```js
+const filter; // A BloomFilter object
+const stream = await client.subscribeToTransactionsWithProofs(filter, { fromBlockHeight: 0 });
+
+stream
+      .on('data', (response) => {
+        const merkleBlock = response.getRawMerkleBlock();
+        const transactions = response.getRawTransactions();
+
+        if (merkleBlock) {
+          const merkleBlockHex = Buffer.from(merkleBlock).toString('hex');
+        }
+
+        if (transactions) {
+          transactions.getTransactionsList()
+          .forEach((tx) => {
```

Comment:
Can you shift it like you would in the actual code? Like so:
```
          transactions.getTransactionsList()
              .forEach((tx) => {
```

Reply:
Doing it right away!